packages/gatsby-theme-docs/gatsby-config.js

```diff
@@ -94,6 +94,14 @@ module.exports = (themeOptions = {}) => {
           path: path.resolve(`./src/content`),
         },
       },
+      // Code samples
+      {
+        resolve: 'gatsby-source-filesystem',
+        options: {
+          name: 'code-examples',
```

Comment:
We use camelCase notation in the other configs

```suggestion
          name: 'codeExamples',
```